src/main/webapp/app/registry/encryption/encryption.component.html

```diff
@@ -0,0 +1,66 @@
+<div>
+    <div class="head">
+        <h2 class="d-inline-block">Encrypt and decrypt properties</h2>
+    </div>
+    <div class="panel panel-default">
+        <div class="panel-body">
+            <ngb-tabset type="pills" class="nav-fill">
```

Comment:
If you add padding to textarea below then tabs should look fine. Try it else leave it ill polish after merge